src/SqlCommonBatchQuery.mli

```diff
@@ -0,0 +1,20 @@
+val query :
+  (
+    sql:string ->
+    ?params:'a ->
+    (
+      [<
+        `Error of 'b
+      | `Select of Js.Json.t array * MySql2.meta
+      ] -> unit
+    ) ->
+    'd
+  ) ->
+  ?batch_size:int ->
+  sql: string ->
+  params:[`Positional of Js.Json.t] ->
+  ([> `Error of 'b | `Select of Js.Json.t array * MySql2.meta] -> unit) ->
```

Comment:
Should this be an open type? From [Real World Ocaml - Chapter 6. Variants][rwo-6-variants]
>The > at the beginning of the variant types above is critical because it marks the types as being open to combination with other variant types. We can read the type [> `Int of string | `Float of float] as describing a variant whose tags include `Int of string and `Float of float, but may include more tags as well. In other words, you can roughly translate > to mean: "these tags or more."

[rwo-6-variants]: https://realworldocaml.org/v1/en/html/variants.html#polymorphic-variants